src/components/common/PageFramework/PageFramework.tsx

```diff
@@ -1,41 +1,59 @@
-import { PropsWithChildren, useEffect } from 'react'
+import React, { PropsWithChildren } from 'react'
 import { useIntl } from 'react-intl'
 import { useLocation } from 'react-router-dom'
+import KalkulatorLogo from '../../../assets/kalkulator.svg'
 
 import { Heading } from '@navikt/ds-react'
 import clsx from 'clsx'
 
 import styles from './PageFramework.module.scss'
 
 export const PageFramework: React.FC<
-  PropsWithChildren & { isFullWidth?: boolean }
-> = ({ children, isFullWidth }) => {
+  PropsWithChildren & {
```

Comment:
Det er kanskje litt pirking, så du får vurdere hva du synes: jeg pleier å bruke boolean convention med has*, eller is*, should* osv, så foreslår noe àla `hasWhiteBg`, og `shouldShowLogo`
(`whiteBg` kunne være en style, en string med farge eller hva som helst, og `showLogo` hørtes ut til meg som en function som man kaller for å vise logoen)